### PR TITLE
[NodeJS] Fixes for UI tests to run

### PR DESCRIPTION
**Pull request by almedina-ms**
Fixes required for making ui tests compile and run during CI builds. 

* Fixes some tests that needed to be updated
* Updates the UI test app to consume the preview 1.6 package

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6847)

### CODE REVIEW

File: source/nodejs/tests/ui-tests/src/testUtils.ts [hunk 1]
```diff
@@ -13,6 +14,7 @@ export class TestUtils {
     async goToTestCase(testCaseName: string): Promise<void> {
         const elementLinkText: Webdriver.WebElement = await this.driver.findElement(Webdriver.By.id(testCaseName));
         await elementLinkText.click();
+        await this.delay(this.delayForAfterCardRendering);
```
Comment (licanhua) [inline source/nodejs/tests/ui-tests/src/testUtils.ts:17]: it's easy to introduce stability issue with `delay`. I prefer to use driver.wait if possible.
https://www.selenium.dev/documentation/webdriver/waits/

 For example:
```
await driver.wait(until.titleIs('webdriver - Google Search'), 1000);
```

File: source/nodejs/tests/ui-tests/src/ui-tests.test.ts [hunk 2]
```diff
@@ -185,19 +194,24 @@ describe("Mock function", function() {
...
-    }), timeOutValueForCarousel);
     test("Test rtl on carousel", (async() => {
         await testUtils.goToTestCase("v1.6/Carousel.rtl");
 
-        let firstCarouselPageVisibility = await testUtils.getElementWithId("firstCarouselPage");
-        Assert.strictEqual(firstCarouselPageVisibility.getAttribute('dir'), 'rtl');
-
-        let secondCarouselPageVisibility = await testUtils.getElementWithId("secondCarouselPage");
-        Assert.strictEqual(secondCarouselPageVisibility.getAttribute('dir'), 'ltr');
+        const firstPage = await testUtils.getElementWithId("firstCarouselPage");
```
Comment (licanhua) [inline source/nodejs/tests/ui-tests/src/ui-tests.test.ts:202]: likely its a 
`for (page of ["firstxx", "secondxx", "thirdxx"]) {
} `

File: source/nodejs/adaptivecards-ui-testapp/src/rendering-utils.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { getTestCasesList } from "./file-retriever-utils";
-import { Action, AdaptiveCard, ExecuteAction, HostConfig, IMarkdownProcessingResult, Input, OpenUrlAction, SubmitAction } from "adaptivecards";
+import { Action, AdaptiveCard, ExecuteAction, HostConfig, IMarkdownProcessingResult, Input, OpenUrlAction, SerializationContext, SubmitAction, Version, Versions } from "adaptivecards";
 import * as Remarkable from "remarkable";
+import { useContext } from "react";
```
Comment (licanhua) [inline source/nodejs/adaptivecards-ui-testapp/src/rendering-utils.ts:6]: you didn't use the hook `useContext`

**Review by licanhua — APPROVED** [review]

**Review by beervoley — APPROVED** [review]
:shipit:

**Review by jwoo-msft — APPROVED** [review]

**Pull request merged**